File: src/test/java/com/everyonewaiter/domain/health/ApkVersionFixture.java
package com.everyonewaiter.domain.health;

import static lombok.AccessLevel.PRIVATE;

import lombok.NoArgsConstructor;

@NoArgsConstructor(access = PRIVATE)
public final class ApkVersionFixture {

  public static ApkVersionCreateRequest createApkVersionCreateRequest() {
    return new ApkVersionCreateRequest(1, 0, 0, "https://cdn.everyonewaiter.com/release.apk");
  }

}

File: src/test/java/com/everyonewaiter/domain/health/ApkVersionFixture.kt
package com.everyonewaiter.domain.health

fun createApkVersionCreateRequest(
  majorVersion: Int = 1,
  minorVersion: Int = 0,
  patchVersion: Int = 0,
  downloadUri: String = "https://github.com/everyonewaiter/everyonewaiter-store-app/releases"
): ApkVersionCreateRequest {
  return ApkVersionCreateRequest(majorVersion, minorVersion, patchVersion, downloadUri)
}
